docs/components/alerts/index.html

```diff
@@ -1,161 +1,114 @@
-            <a href="#">Acknowledge</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
+    <h2 id="default">Demos</h2>
+    <div id="vue-alertDemo" class="hxRow" v-cloak>
+      <div class="hxCol hxSpan-12-xs hxSpan-3-lg hxOrder-2-lg">
+        <h3>Options</h3>
+        <p>
+          <label>
+            <input type="checkbox" v-model="isClosable">
+            Closable
+          </label>
+        </p>
+        <p>
+          <b>Variant:</b><br />
+          <select v-model="variant">
+            <option v-for="_variant in variants" :value="_variant">
+              {% raw %}{{ _variant.label }}{% endraw %}
+            </option>
+          </select>
+        </p>
       </div>
-    </div>
-    {% code 'html' %}
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert">
-          <div class="hxAlert__icon">
-            <hx-icon type="info-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Info</span>
-            Here is helpful information.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Acknowledge</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
-      </div>
-    {% endcode %}
-  </section>
-
-  <section>
-    <h2 id="error">Error</h2>
 
-    <div class="demo">
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert hxAlert--error">
-          <div class="hxAlert__icon">
-            <hx-icon type="exclamation-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Error</span>
-            Change a few things up and try submitting again.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Correct Issue</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
-      </div>
-    </div>
-    {% code 'html' %}
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert hxAlert--error">
-          <div class="hxAlert__icon">
-            <hx-icon type="exclamation-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Error</span>
-            Change a few things up and try submitting again.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Correct Issue</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
+      <div class="hxCol hxSpan-12-xs hxSpan-9-lg hxOrder-1-lg">
+        <h3>Basic Alert</h3>
+        <div class="demo alert-demo">
+          <hx-disclosure :disabled="!isClosable" class="hxBtn" aria-controls="myAlert">
+            Toggle Alert
```

Comment:
the demo looks super good! just wondering what the thoughts were for toggling the alert in the demo, is this necessary?

Reply:
I wanted to demonstrate that alerts could be closed, so I added the toggle button to show them again after being closed. It doesn't necessarily need to be a toggle button, but that's what I used.